.github/workflows/build-publish-next.yml

```diff
@@ -39,6 +39,7 @@ jobs:
     - name: Build and publish images
       run: |
         docker image prune -a -f
+        export GITHUB_TOKEN=${{ secrets.CHE_BOT_GITHUB_TOKEN }}
```

Comment:
`GITHUB_TOKEN` secret is created automatically
https://docs.github.com/en/actions/reference/authentication-in-a-workflow#about-the-github_token-secret

Reply:
yes, but only when the secret == secrets.GITHUB_TOKEN.

Here, we're using a different secret name, secrets.CHE_BOT_GITHUB_TOKEN.

But if you don't think we need this, you can close the PR.

Reply:
it's better to use the generated token as we have a fresh one for each run (while we could decrease quite a lot the one from che-bot)